src/components/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+/* eslint-disable */
+import calculate from '../logic/calculate';
+/* eslint-enable */
```

Comment:
if you had used a disable-next-line there, then there would be no need to call enable again